setup.py

```diff
@@ -33,13 +37,24 @@
 csr_file = 'kms-signing.csr'
 config_file = 'config.json'
 
-kms = boto3.client('kms')
+
+if run_mode == 'DEV':
```

Comment:
Uses env file config

Reply:
The neat thing about that is that even if someone would want something else then the "default" AWS config, they could configure dev mode and put their config/credentials in the env file